src/AutoRest.CSharp/Generation/Writers/RestClientWriter.cs

```diff
@@ -243,6 +244,18 @@ private void WriteRequestCreation(CodeWriter writer, RestClientMethod clientMeth
                             flattenedSchemaRequestBody.Serialization,
                             w => w.Append(modelVariable));
                         break;
+                    case UrlEncodedBody urlEncodedRequestBody:
+                        WriteHeaders(writer, clientMethod, request, content: true);
+                        writer.Append($"var contentBuilder = new Azure.Core.FormUrlEncodedContent.Builder ();\n");
+
+                        foreach (var (name, value) in urlEncodedRequestBody.Values)
+                        {
+                            writer.Append($"contentBuilder.Add(\"{name}\",");
```

Comment:
Use literal syntax for the name:

![image](https://user-images.githubusercontent.com/1697911/108411585-ab43f200-71dd-11eb-8910-4d9e31b41eed.png)

Reply:
I'm not tracking you here. I harvest the name in RestClient.cs using GetRequestParameterName(bodyRequestParameter), and store the string (name is a string here).

```
        private string GetRequestParameterName (RequestParameter requestParameter)
        {
            string defaultName = requestParameter.Language.Default.Name;
            return requestParameter.Language.Default.SerializedName ?? defaultName;
        }
```